### PR TITLE
fix(slider): high contrast improvements for filled track

### PR DESCRIPTION
**Pull request by mdt2**
## Description

- Uses Spectrum CSS Slider version 4.2.1

## Related issue(s)

- Spectrum CSS [PR 2325](https://github.com/adobe/spectrum-css/pull/2325)

## Motivation and context

Add Windows High Contrast Mode improvements for sliders with a filled track.

## How has this been tested?

- [ ] Test that Sliders are rendering correctly on the docs page
- [ ] Test that Sliders are rendering correctly in Storybook
- [ ] VRTs are passing

## Screenshots
Before:
Screen Shot 2024-01-18 at 4 35 34 PM

After:
Screen Shot 2024-01-18 at 4 33 30 PM

## Types of changes

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [ ] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [ ] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Comment by Westbrook** [comment]
Yep, the changes look like what I'd expect from you change. Can you take a look at [this part](https://github.com/adobe/spectrum-web-components?tab=readme-ov-file#keeping-ci-assets-updated) of the docs to get the cache updated?

**Comment by mdt2** [comment]
@Westbrook thanks for that link! I updated the cache key, and after pushing realized I'm out of sync with main so I'll need to rebase, but before I do I'm now seeing that some Toast stories are removed [in the VRTs](https://0ac43c1a708ab151a6b2ace9e1834b91--spectrum-web-components.netlify.app/review/#ToastStories/Info.png) which is definitely unexpected for this one. Any idea what could be causing that?

**Comment by Westbrook** [comment]
> Any idea what could be causing that?

Sometimes failures cause failures in the VRTs and you need to update the cache value twice for it to go back to clean. Sadly.

**Comment by mdt2** [comment]
> Sometimes failures cause failures in the VRTs and you need to update the cache value twice for it to go back to clean. Sadly.

Ok thanks. I'll move forward with updating the cache value to the new one

File: packages/slider/package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
         "@spectrum-web-components/theme": "^0.40.3"
     },
     "devDependencies": {
-        "@spectrum-css/slider": "^4.1.19"
+        "@spectrum-css/slider": "4.2.1-beta.0"
```
Comment (Westbrook) [inline packages/slider/package.json:86]: Have this gotten a full release?

**Review by Westbrook — COMMENTED** [review]
Sorry, I rebased this thinking it was ready to go, but we need to update to a non-beta release before we can merge. Keep an eye out for that when updating the version. After that, we should be good to go here. Thanks for taking care of this!

File: packages/slider/package.json (hunk 1 above)
Comment (mdt2) [reply packages/slider/package.json:86]: Not yet, on our side [this work](https://github.com/adobe/spectrum-css/pull/2325) is pending re-review now that we've validated the changes here. I'll reach out to the team again to try to get some eyes on it.

**Comment by Westbrook** [comment]
Would love to get a stable release on this if we're ready.

**Comment by mdt2** [comment]
> Would love to get a stable release on this if we're ready.

@Westbrook we were able to get this merged and released today and I've updated the PR with the new version 👍

**Review by Westbrook — APPROVED** [review]
Thanks for driving this home. Looking forward to including it in this week's release.

**Pull request merged**